base/resources/landing_page/script.js

```diff
@@ -0,0 +1,98 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var launcherLinks = document.getElementsByClassName("launcherlink");
+for (var i = 0; i < launcherLinks.length; i++) {
+  launcherLinks[i].setAttribute('href',window.location.pathname + launcherLinks[i].getAttribute('data-path'));
+}
+});
+
+
+// document.getElementById("page-path-debug").innerHTML = "Page path is: " + window.location.pathname
+
+//accessPortModal includes the lightbox
+var accessPortModal = document.getElementById("access-port-modal");
+var accessPortButton = document.getElementById("access-port-button");
+var closeModal = document.getElementById("close-access-port-modal");
+
+accessPortButton.onclick = function() 
+{
+  accessPortModal.style.display = "block";
+  document.getElementById("port-input").focus();
+
+  //Remove tab focus from launcher buttons
+  var launchers = document.querySelectorAll(".launcher, .launcherlink");
+  var i = launchers.length;
+  while (i--)
+  {
+    launchers[i].setAttribute("tabindex", "-1");
+  }
+}
+
+accessPortButton.onkeypress = function(event) 
+{
+  if (event.code === 'Enter')
+  {
+    accessPortModal.style.display = "block";
+    document.getElementById("port-input").focus();
+  } 
+}
+
+closeModal.onclick = function() 
+{
+  closeAccessPortModal();
+}
+
+window.onclick = function(event) 
+{
+  //Click lightbox outside of modal-content
+  if (event.target == accessPortModal) 
+  {
+    closeAccessPortModal();
+  }
+} 
+
+function accessPortInput() 
+{
+    var port = document.getElementById("port-input").value;
+    if(port)
+    {
+        if (isNaN(port) || port < 1 || port > 655335) 
```

Comment:
Can probably increase lower bound to 1024. Ports less than that are privileged and non-root can't (shouldn't!) be able to open those.